src/CaptureFile/CaptureFileConstants.h

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef CAPTURE_FILE_CONSTANTS_H_
+#define CAPTURE_FILE_CONSTANTS_H_
+
+#include <stddef.h>
+#include <stdint.h>
+
+#include <array>
+
+constexpr size_t kFileSignatureSize = 4;
+constexpr const char* kFileSignature = "ORBT";
```

Comment:
ORBIT?

Reply:
Okay, that is fine, but could you then add a comment?

Reply:
done.